src/components/MemoryItem.js

```diff
@@ -2,39 +2,33 @@ import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
 
 MemoryItem.propTypes = {
+  onDetail: PropTypes.func.isRequired,
   image: PropTypes.node,
   subtitle: PropTypes.string,
 }
 
-export default function MemoryItem({ image, subtitle }) {
+export default function MemoryItem({ image, subtitle, onDetail }) {
   return (
-    <ListWrapper>
-      <li>
-        <Image src={image} alt="" width="100%" />
-        <span>{subtitle}</span>
-      </li>
-    </ListWrapper>
+    <Wrapper onClick={() => onDetail(image)}>
+      <Image src={image} alt="" width="100%" />
```

Comment:
An image should always get both, width and height attributes, but always without any unit. It's only to give the browser the expected aspect ratio. Please have a look at the Article that Mel shared today in _hh-web-21-3_